src/bridges/layerZero/IexecLayerZeroBridge.sol

```diff
@@ -137,18 +137,18 @@ contract IexecLayerZeroBridge is
 
     /**
      * @notice Indicates whether the OFT contract requires approval to send tokens
-     * @return requiresApproval Always returns false for this implementation
+     * @return requiresApproval Returns true if deployed on Ethereum, false otherwise
      */
-    function approvalRequired() external pure virtual returns (bool) {
-        return false;
+    function approvalRequired() external view virtual returns (bool) {
+        return block.chainid == 1;
```

Comment:
This is very limited as we cannot use it on Arbitrum for eg. (say to bridge from Arbitrum to another L2).
It's better to use a configuration variable set at construction.